tests/fast-integration/skr-svcat-migration-test/test-helpers.js

```diff
@@ -68,6 +69,89 @@ async function installBTPServiceOperatorMigrationHelmChart() {
     }
 }
 
+async function getFunctionPod(functionName) {
+    let labelSelector = `serverless.kyma-project.io/function-name=${functionName},serverless.kyma-project.io/resource=deployment`;
+    let res = {};
+    for (let i = 0; i < 30; i++) {
+        res = await listPods(labelSelector);
+        if (res.body.items.length == 1) {
+            let pod = res.body.items[0];
+            if (pod.status.phase == "Running") {
+                return pod
+            }
+        }
+        sleep(10000);
+    }
+    if (res.body.items.length != 1) {
+        let podNames = res.body.items.map(p=>p.metadata.name);
+        let phases = res.body.items.map(p=>p.status.phase);
+        throw new Error(`Failed to find function ${functionName} pod in 5 minutes. Expected 1 ${labelSelector} pod with phase "Runninng" but found ${res.body.items.length}, ${podNames}, ${phases}`);
+    }
+}
+
+async function checkPodPresetEnvInjected() {
+    let cmd = `for v in {vars}; do x="$(eval echo \\$$v)"; if [[ -z "$x" ]]; then echo missing $v env variable; exit 1; else echo found $v env variable; fi; done`;
+    for (f of functions) {
+        let pod = await getFunctionPod(f.name);
+        let envCmd = cmd.replace("{vars}",f.checkEnvVars);
+        await kubectlExecInPod(pod.metadata.name, "function", ["sh", "-c", envCmd]);
+    }
+}
+
+function sleep(ms) {
```

Comment:
We already have such function ([here](https://github.com/kyma-project/kyma/blob/main/tests/fast-integration/utils/index.js#L72))

Reply:
thanks, missed it. Will fix right away :)